src/dashboard/src/templates/archival_storage/list.html

```diff
@@ -96,7 +96,11 @@
       <tbody>
         {% for item in aips %}
           <tr>
-            <td><a href="{% url 'view_aip' item.uuid %}">{{ item.name }}</a></td>
+              {% if item.encrypted %}
+              <td><a href="{% url 'view_aip' item.uuid %}">{{ item.name }} ({% trans "encrypted" %})</a></td>
```

Comment:
Oh I didn't see the parens! Sorry about that! `{{ aip_name }} (encrypted)` looks better to me but I think both ways work, up toyou.